### DISCUSSION TITLE
Towing and sling loading not working

### DISCUSSION THREAD

**Discussion by FrozenScarlett** [Vdauphin/HeartsAndMinds#1128] [Question and answer]
I am having an issue where the hook option is not appearing for wrecks when using a chinook or other aircraft. We have also tried to use the tow function on a UH-1Y, with a flat bed and was unable to interact on that aircraft either. is there a fix for this?

**Comment by Vdauphin** [comment]
Hello,
You can't tow aircraft (plane and helicopter). You can only sling load them: http://vdauphin.github.io/HeartsAndMinds/InGame-documentation#sling-loading

> is there a fix for this?

You can customize this behavior here https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/mission.sqf#L575-L588
Basically remove `"Helicopter"`
Cheers

**Reply by FrozenScarlett** [reply]
I have tried to slingload helis they are not giving me the hook option, and when you try to use adv sling loading they dont connect to anything just the ground. its acting almost as if the entity does exist, for instance you can walk right up a helicopter thats blown up, but cant interact on it at all

**Reply by Vdauphin** [reply]
> I have tried to slingload helis they are not giving me the hook option, 

Please follow this http://vdauphin.github.io/HeartsAndMinds/InGame-documentation#sling-loading

> and when you try to use adv sling loading they dont connect to anything just the ground. 

H&M doesn't have adv sling loading, it only use =BTC= Lift and A3 sling loading, 

> its acting almost as if the entity does exist, for instance you can walk right up a helicopter thats blown up, but cant interact on it at all

Looks like something is wrong on your side

**Reply by FrozenScarlett** [reply]
So its not compatable with ADV slingloading? that sounds like the issue what is A3 slingloading?

**Reply by Vdauphin** [reply]
> So its not compatable with ADV slingloading?

I don't know, ask to the mod author

> that sounds like the issue what is A3 slingloading?

It is just the vanilla system. BI made a tutotrial 
https://www.youtube.com/watch?v=a6FCYbVTFqg 
https://armedassault.fandom.com/wiki/Sling_Loading_(showcase)
